recipes/install.rb

```diff
@@ -7,11 +7,33 @@
 
 include_recipe "java"
 
+
+group node['hops']['group'] do
```

Comment:
Could you please explain why you need to create the `hdfs` user here?

Reply:
it happened that flink::install ran before hops::install and the user did not exist. I will move the code to using the `flink` user anyway.